store/tikv/txn.go

```diff
@@ -283,6 +284,13 @@ func (txn *KVTxn) SetEnable1PC(b bool) {
 	txn.enable1PC = b
 }
 
+// SetCasualConsistency indicates if the transaction does not need to
+// guarantee linearizability. Default value is false which means
+// linearizability is guaranteed.
+func (txn *KVTxn) SetCasualConsistency(b bool) {
```

Comment:
```suggestion
func (txn *KVTxn) SetCausalConsistency(b bool) {
```

Reply:
There are 2 names which indicate the same thing:`GuaranteeLinearizability` == not `CausalConsistency`. Why don't we use consistent names?

Reply:
I think the purpose is to let `false` mean linearizability (which is the safer and default option). If the field is uninitialized (thus is `false`), the behavior is default.

Reply:
true. Use default value as the default behaviour is easier to reason correctness.